### PR TITLE
Update admin email

### PR DESCRIPTION
**Pull request by nicholasmueller**
Issue/Task Number: [949](https://github.com/omisego/ewallet/issues/949)

# Overview

- update admin email in myprofile page
- link in verify email takes you to a page that makes the verify call with button back to dashboard

- redirect url still has /api in route. but if i remove this from my config, i get the cors error again. @jarindr knows more?

# Implementation Details

- split avatar, password, email updates into their respective calls as previously avatar update was dependent on email update. api calls also updated

### CODE REVIEW

File: apps/admin_panel/assets/src/services/currentUserService.js [hunk 1]
```diff
@@ -13,9 +15,12 @@ export function getCurrentUserAccount () {
...
 
-export function updateCurrentUser ({ email }) {
+export function updateCurrentUserEmail ({ email }) {
   return authenticatedRequest({
-    path: 'me.update',
-    data: { email }
+    path: 'me.update_email',
+    data: {
+      email,
+      redirect_url: `${ADMIN_API_URL}verify_email?email={email}&token={token}`
```
Comment (jarindr) [inline apps/admin_panel/assets/src/services/currentUserService.js]: `ADMIN_API_URL` is an API url, we should use something like
`root_domain/verify_email?email={email}&token={token}`

for Example, 
```js
 redirectUrl: window.location.href.replace(
          this.props.location.pathname,
          '/create-new-password/'
```

Comment (jarindr) [reply apps/admin_panel/assets/src/services/currentUserService.js]: ref: `ewallet/apps/admin_panel/assets/src/omg-forgetPassword-form/index.js`

File: apps/admin_panel/assets/src/omg-page-verify-email/index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  const { email, token } = queryString.parse(search)
+  const [ pageState, setPageState ] = useState('loading')
+
+  useEffect(() => {
+    doVerifyEmail()
+  }, [])
+
+  const doVerifyEmail = async () => {
+    const res = await verifyEmail({ email, token })
+    console.log('res: ', res)
```
Comment (jarindr) [inline apps/admin_panel/assets/src/omg-page-verify-email/index.js]: log!

**Comment by nicholasmueller** [comment]
@jarindr made changes, please see again

**Pull request merged**